web/src/main/java/org/cbioportal/web/DataAccessTokenController.java

```diff
@@ -129,13 +125,8 @@ private String getAuthenticatedUser(Authentication authentication) {
         return username;
     }
 
-    private DataAccessToken createDataAccessToken(String userName, Boolean myAllowRevocationOfOtherTokens) {
-        DataAccessToken token;
-        if (myAllowRevocationOfOtherTokens != null) {
-            token = tokenService.createDataAccessToken(userName, myAllowRevocationOfOtherTokens);
-        }  else {
-            token = tokenService.createDataAccessToken(userName);
-        }
+    private DataAccessToken createDataAccessToken(String userName) {
```

Comment:
This function no longer serves any purpose. I suggest we drop this function and replace calls to it with a call to tokenService.createDataAccessToken(userName) instead.